Cargo.toml

```diff
@@ -24,13 +24,13 @@ incrementalmerkletree = { version = "0.8.2", features = ["legacy-api"] }
 orchard = "0.11"
 sapling-crypto = "0.5"
 zcash_address = "0.9"
-zcash_client_backend = "0.19"
+zcash_client_backend = "0.20"
```

Comment:
I though that zebra no longer needed the `zcash_client_backend` dependency?

Reply:
Indeed, it seems that this was just not removed at the time, and is now superfluous.